Autopilot/Src/gimbal.cpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by anthony Luo on 6/19/2021.
+//
+
+#include "Interchip_A.h"
+#include <gimbal.hpp>
+
+static const float gimbalMaxPitch = 3.14159;
+static const float gimbalMaxYaw = 3.14159/2;
```

Comment:
So the ranges are:

pitch: 0 to PI

Yaw: -PI/2 to PI/2

Just confirming for when I do value validation :))

Reply:
Also please use our PI constant here

Reply:
> So the ranges are:
> 
> pitch: 0 to PI
> 
> Yaw: -PI/2 to PI/2
> 
> Just confirming for when I do value validation :))

yes, pitch goes from 0 to PI, yaw goes from -PI/2 to PI/2. Pitch is supposed to be straight forward (level with airframe) at 0 and straight backward (level with airframe) at 180 (this will change depending on the orientation of the servos at startup). Yaw goes from -pi/2 straight left to pi/2 straight right.